src/types.js

```diff
@@ -385,7 +385,8 @@ export type LeakType = {
 };
 
 export type MaterializeType = {
-  materialize(realm: Realm, object: ObjectValue): void,
+  materializeObject(realm: Realm, object: ObjectValue): void,
```

Comment:
Why even expose the single-object materialization? Doesn't seem to be needed, and nobody should call that, right?

Reply:
Eventually it should go away. Right now, it is dangling out of the leaking implementation.